src/ray/core_worker/transport/direct_actor_transport.h

```diff
@@ -416,8 +416,8 @@ class ActorSchedulingQueue : public SchedulingQueue {
         pool_(pool),
         is_asyncio_(is_asyncio) {
     if (is_asyncio_) {
-      RAY_LOG(INFO) << "Setting actor as async with max_concurrency=" << max_concurrency
-                    << ", creating new fiber thread.";
+      RAY_LOG(DEBUG) << "Setting actor as async with max_concurrency=" << max_concurrency
```

Comment:
This log only prints once when initialized and max_concurrency is helpful(i think), maybe INFO is ok?

Reply:
sgtm.